Lombiq.Tests.UI/Services/UITestContext.cs

```diff
@@ -76,6 +77,15 @@ public class UITestContext
         Justification = "Deliberately modifiable by consumer code.")]
     public Dictionary<string, object> CustomContext { get; } = new();
 
+    /// <summary>
+    /// Gets a dictionary storing some custom data for collecting in failure dump.
+    /// </summary>
+    [SuppressMessage(
+        "Design",
+        "MA0016:Prefer return collection abstraction instead of implementation",
+        Justification = "Deliberately modifiable by consumer code.")]
```

Comment:
What does this mean? If you used `IDictionary<string, Func<Task<Stream>>>` that's still modifiable.